packages/mask/src/plugins/Gitcoin/SNSAdaptor/index.tsx

```diff
@@ -42,8 +42,8 @@ const sns: Plugin.SNSAdaptor.Definition = {
         {
             ApplicationEntryID: base.ID,
             category: 'dapp',
-            description: <Trans i18nKey="plugin_gitcoin_description" />,
-            name: <Trans i18nKey="plugin_gitcoin_name" />,
+            description: <Trans ns={PluginId.Gitcoin} i18nKey="description" />,
```

Comment:
Those two fields accepts I18NField. Please use i18nField (`{ i18nKey: '__description', fallback: '...' }`)

Reply:
`description` for Gitcoin plugin is so long, it will mess up the code.

```json
    "description": "Display specific information of Gitcoin projects, donate to a project directly on social media.",
```

```diff
-           description: <Trans ns={PluginId.Gitcoin} i18nKey="description" />,
+           description: {
+               i18nKey: 'description',
+               fallback:
+                   'Display specific information of Gitcoin projects, donate to a project directly on social media.',
+           },
```

BTW, the `fallback` can easily outdate without being noticed. And managing text/translation in too many places looks like not following the DRY principle.

Reply:
Agree. Track this issue on #6357.